packages/gatsby-image/src/index.js

```diff
@@ -546,7 +552,7 @@ class Image extends React.Component {
                 src={image.src}
                 crossOrigin={this.props.crossOrigin}
                 srcSet={image.srcSet}
-                style={imageStyle}
+                style={{ imageStyle }}
```

Comment:
I'm not sure this is correct

Reply:
@smashercosmo hmm agreed, not sure where this is coming from actually. . . I didn't update this intentionally.  Pushing fix.